api/grpcserver/mesh_service.go

```diff
@@ -357,15 +361,27 @@ func (s MeshService) readLayer(layer *types.Layer, layerStatus pb.Layer_LayerSta
 }
 
 // LayersQuery returns all mesh data, layer by layer
-func (s MeshService) LayersQuery(ctx context.Context, in *pb.LayersQueryRequest) (*pb.LayersQueryResponse, error) {
+func (s MeshService) LayersQuery(_ context.Context, in *pb.LayersQueryRequest) (*pb.LayersQueryResponse, error) {
```

Comment:
in others places you just didn't include _

Reply:
if `in` is used, i.e. we need one named argument, then I think all of the arguments need names, right? that's why some include `_` and some don't

Reply:
you are right, I did a quick check on go playground but it yielded false result because I wrote it like this 
`` func F(int, x int) {}
   F(1,1)
```
now this worked because the compiler treated the first `int` as the name of the first parameter.

Reply:
wow that's funky! I didn't even know that was possible. good to know.